app/com/arpnetworking/metrics/portal/reports/ReportRepository.java

```diff
@@ -45,5 +49,23 @@
      * @param organization The {@code Organization} which owns the report.
      */
     void addOrUpdateReport(Report report, Organization organization);
+
+    /**
+     * Query reports.
+     *
+     * This is a specialization of {@link JobRepository#query} for cases where you want to
+     * handle the reports themselves, outside of a Job context.
+     *
+     * @param query The {@code JobQuery} instance to execute.
+     * @return The reports resulting from executing the query.
+     */
+    QueryResult<Report> queryReports(JobQuery<Report.Result> query);
+
+    @Override
+    default QueryResult<Job<Report.Result>> query(JobQuery<Report.Result> query) {
+        final QueryResult<Report> reports = queryReports(query);
+        // This re-wrap step is necessary to satisfy the compiler typecheck.
+        return new DefaultQueryResult<>(reports.values(), reports.values().size());
+    }
```

Comment:
Should there also be a `createQuery` method?

Reply:
Yes, you're right I missed that.